src/relay_pic/relay_general.h

```diff
@@ -0,0 +1,37 @@
+#ifndef RELAY_GENERAL_H
+#define RELAY_GENERAL_H
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+uint16_t analog_inputs[2];

```

Comment:
Unless you want to create new instances of this variable in each compilation unit that includes this header, this should be declared `extern`

Reply:
Perhaps use a getter function instead, such as `get_analog_inputs()`

Reply:
Resolved by switching to getter, needs testing so I am blocking until I can find time to do so.